lakefile.lean

```diff
@@ -10,3 +10,6 @@ lean_lib «TensorLib» where
 @[default_target]
 lean_exe "tensorlib" where
   root := `Main
+
+require mathlib from git
+  "https://github.com/leanprover-community/mathlib4"
```

Comment:
Specify tag corresponding to the version of lean we're using

Reply:
Good idea. Done.